src/main/kotlin/no/nav/syfo/dialogmote/database/ArbeidstakersvarPublishedToKafkaAtQueries.kt

```diff
@@ -0,0 +1,55 @@
+package no.nav.syfo.dialogmote.database
+
+import no.nav.syfo.application.database.toList
+import no.nav.syfo.dialogmote.api.domain.*
+import no.nav.syfo.dialogmote.domain.DialogmoteSvarType
+import no.nav.syfo.domain.PersonIdentNumber
+import no.nav.syfo.util.toOffsetDateTimeUTC
+import java.sql.*
+import java.time.*
+import java.util.*
+
+const val queryGetUnpublishedArbeidstakersvar =
+    """
+        SELECT m.uuid as mote_uuid, ma.personident, v.svar_type,  v.uuid as varsel_uuid, v.created_at as brev_sent_at, v.svar_tidspunkt as svar_received_at
+        FROM motedeltaker_arbeidstaker_varsel v
+        INNER JOIN motedeltaker_arbeidstaker ma ON v.motedeltaker_arbeidstaker_id = ma.id
+        INNER JOIN mote m ON ma.mote_id = m.id
+        WHERE v.svar_tidspunkt IS NOT NULL
+        AND v.svar_published_to_kafka_at IS NULL
+        LIMIT 100;
+    """
+
+fun Connection.getUnpublishedArbeidstakersvar(): List<Arbeidstakersvar> {
+    return this.prepareStatement(queryGetUnpublishedArbeidstakersvar).use {
+        it.executeQuery().toList { toArbeidstakersvar() }
+    }
+}
+
+const val queryUpdateArbeidstakerVarselPublishedAt =
+    """
+        UPDATE motedeltaker_arbeidstaker_varsel
+        SET svar_published_to_kafka_at = ?
+        WHERE uuid = ?
+    """
+
+fun Connection.updateArbeidstakerVarselPublishedAt(
+    varseluuid: UUID,
+) {
+    val now = Timestamp.from(Instant.now())
+    this.prepareStatement(queryUpdateArbeidstakerVarselPublishedAt).use {
+        it.setTimestamp(1, now)
+        it.setString(2, varseluuid.toString())
+        it.execute()
+    }
```

Comment:
Her kan det være lurt å kaste en exception hvis antall oppdaterte rader er != 1 (se eksempler for andre update-queries i app'en).